sql/core/src/test/scala/org/apache/spark/sql/execution/command/v2/AlterTableRenameSuite.scala

```diff
@@ -34,4 +34,15 @@ class AlterTableRenameSuite extends command.AlterTableRenameSuiteBase with Comma
       }
     }
   }
+
+  test("include catalog in the destination table") {
+    withNamespaceAndTable("ns", "dst_tbl", catalog) { dst =>
+      val src = dst.replace("dst", "src")
+      sql(s"CREATE TABLE $src (c0 INT) $defaultUsing")
+      sql(s"INSERT INTO $src SELECT 0")
+
+      sql(s"ALTER TABLE $src RENAME TO $catalog.ns.dst_tbl")
```

Comment:
Does it support rename to a different catalog?

Reply:
Thanks for checking this. I added the different catalog case and pushed it.